src/strategies/plearn/index.ts

```diff
@@ -0,0 +1,214 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
+
+export const author = 'plearnclub';
+export const version = '0.0.1';
+
+const lockedPoolabi = [
+  {
+    inputs: [
+      {
+        internalType: 'address',
+        name: '',
+        type: 'address'
+      }
+    ],
+    name: 'userInfo',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: 'amount',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
+
+const foundingInvestorPoolabi = [
+  {
+    inputs: [
+      {
+        internalType: 'address',
+        name: '',
+        type: 'address'
+      }
+    ],
+    name: 'userInfo',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: 'initialAmount',
+        type: 'uint256'
+      },
+      {
+        internalType: 'uint256',
+        name: 'amount',
+        type: 'uint256'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
+
+const pendingWithdrawalabi = [
+  {
+    inputs: [
+      {
+        internalType: 'address',
+        name: 'user',
+        type: 'address'
+      }
+    ],
+    name: 'lockedBalances',
+    outputs: [
+      {
+        internalType: 'uint256',
+        name: 'total',
+        type: 'uint256'
+      },
+      {
+        internalType: 'uint256',
+        name: 'unlockable',
+        type: 'uint256'
+      },
+      {
+        internalType: 'uint256',
+        name: 'locked',
+        type: 'uint256'
+      },
+      {
+        components: [
+          {
+            internalType: 'uint256',
+            name: 'amount',
+            type: 'uint256'
+          },
+          {
+            internalType: 'uint256',
+            name: 'unlockTime',
+            type: 'uint256'
+          }
+        ],
+        internalType: 'struct PendingWithdrawal.LockedBalance[]',
+        name: 'lockData',
+        type: 'tuple[]'
+      }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  const score = await erc20BalanceOfStrategy(
+    space,
+    network,
+    provider,
+    addresses,
+    options,
+    snapshot
+  );
+
+  const lockedPoolBalances = await Promise.all(
+    options.lockedPoolAddresses.map((item) =>
+      multicall(
+        network,
+        provider,
+        lockedPoolabi,
+        addresses.map((address: any) => [
+          item.address,
+          'userInfo',
+          [address],
+          { blockTag }
+        ]),
+        { blockTag }
+      )
+    )
+  );
```

Comment:
Is it possible to limit the number of `lockedPoolAddresses` `foundingInvestorPoolAddresses`, `pendingWithdrawalAddresses` and make sure number of requests doesn't cross 5 https://github.com/snapshot-labs/snapshot-strategies#code

Reply:
@jeeraponnnnnnnn

Reply:
@ChaituVR 
I've implemented the changes as you suggested. Now, I've limited the arrays for lockedPoolAddresses, foundingInvestorPoolAddresses, and pendingWithdrawalAddresses to have no more than 5 addresses each. Thanks

Reply:
But that will max 15 network calls, we can only allow upto max 5 network calls https://github.com/snapshot-labs/snapshot-strategies#code

Reply:
@ChaituVR  I've updated code, could you please take another look at it?